_i18n/en.yml

```diff
@@ -36,17 +70,29 @@ long_month:
   November: "November"
   December: "December"
 
-## Generic strings
-generic:
-  download: "Download"
-  contribute: "Contribute"
-  view: "View All"
-  learn: "Learn More"
-  share: "Share"
-  error: "Error"
-  redirect: "Redirect"
-  subscribe: "Subscribe"
+language:
+  select: "Language"
```

Comment:
Check: user where?

Reply:
`grep -r "language.select" *` didn't yield any result, so I think I had added it for something while trying things, but ended up not using it.